ocaml/xapi/rbac.ml

```diff
@@ -244,7 +244,7 @@ let assert_permission ~__context ~permission =
   assert_permission_name ~__context ~permission:permission.role_name_label
 
 (* this is necessary to break dependency cycle between rbac and taskhelper *)
-let init_task_helper_rbac_has_permission_fn =
+let _init_task_helper_rbac_has_permission_fn =
```

Comment:
Does this mean this function is unused and can be removed?

Reply:
Leaving it only can lead to confusion, please delete it. We have git to remember the function